src/script/interpreter.h

```diff
@@ -133,6 +137,13 @@ bool CheckSignatureEncoding(const std::vector<unsigned char> &vchSig, unsigned i
  */
 bool CheckDataSignatureEncoding(const std::vector<uint8_t> &vchSig, uint32_t flags, ScriptError *serror);
 
+/**
+ * Check that the signature provided to authentify a transaction is properly
+ * encoded Schnorr signature (or null). Signatures passed to the new-mode
+ * OP_CHECKMULTISIG and its verify variant must be checked using this function.
+ */
+bool CheckTransactionSchnorrSignatureEncoding(const std::vector<uint8_t> &vchSig, uint32_t flags, ScriptError *serror);
+
```

Comment:
probably not needed but shouldn't we add also `CheckSignatureEncodingSigHashChoice` declaration here?

Reply:
that's a static helper function.